src/main/java/ToDo.java

```diff
@@ -0,0 +1,17 @@
+public class ToDo extends Task{
+    String tag = "[T]";
+    public ToDo() {
+        super.tag = tag;
+    }
+
+    @Override
+    public void genDscp(String input) throws InvalidTodo{
+        String dscp = input.replace("todo ", "");
+        if (dscp.isBlank()) {
+            throw new InvalidTodo();
+        }
+        super.task = dscp;
+    }
+
+    //Override toString
```

Comment:
Might want to make the comment into a TODO: ... to indicate as a yet to implement method

Reply:
I see, thanks for the tip!